_posts/2017-05-23-0.1-announcement.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Introducing Istio: A robust service mesh for microservices"
+overview: Istio 0.1 announcement
+published: true
+permalink: blog/istio-service-mesh-for-microservices.html
```

Comment:
This is the permalink we've already told our partners we're using, so it is somewhat fixed already.